### PR TITLE
scripts/examples: Add imxrt board control examples.

### PR DESCRIPTION
**Pull request by kwagyeman**
Fixes: https://github.com/openmv/openmv/issues/2071

### CODE REVIEW

**Comment by iabdalkader** [comment]
This pull request has been mentioned on **OpenMV Forums**. There might be relevant details there:

https://forums.openmv.io/t/openmv-cam-rt1062-examples-and-pyb/9217/2

File: scripts/examples/50-OpenMV-Boards/50-IMXRT-Boards/50-Board-Control/spi_control.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
+# This work is licensed under the MIT license.
+# Copyright (c) 2013-2024 OpenMV LLC. All rights reserved.
+# https://github.com/openmv/openmv/blob/master/LICENSE
+#
+# SPI Control
+#
+# This example shows how to use the SPI bus on your OpenMV Cam to directly
+# the LCD shield without using the built-in lcd shield driver. You will need
+# the LCD shield to run this example.
```
Comment (iabdalkader) [inline scripts/examples/50-OpenMV-Boards/50-IMXRT-Boards/50-Board-Control/spi_control.py:9]: Typo.

```suggestion
# This example shows how to use the SPI bus to directly control the LCD shield without
# using the built-in LCD driver. You will need the LCD shield to run this example.
```

File: scripts/examples/50-OpenMV-Boards/50-IMXRT-Boards/50-Board-Control/spi_loopback.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    var = bytearray(5)
+    s = "%05d" % i
+    spi.write_readinto(bytes(s, 'utf-8'), var)
+    print(var)
+
+    time.sleep_ms(50)
+
+    i += 1
+    if (i >= 256):
+        i = 0
```
Comment (iabdalkader) [inline scripts/examples/50-OpenMV-Boards/50-IMXRT-Boards/50-Board-Control/spi_loopback.py:27]:
```suggestion
```

File: scripts/examples/50-OpenMV-Boards/50-IMXRT-Boards/50-Board-Control/uart_loopback.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+# Always pass UART 1 for the UART number for your OpenMV Cam.
+# The second argument is the UART baud rate.
+uart = UART(1, 19200, timeout_char=200)
+
+while True:
+    uart.write("Hello World!\r")
+    l = uart.readline()
+    print(l)
+    time.sleep_ms(100)
```
Comment (iabdalkader) [inline scripts/examples/50-OpenMV-Boards/50-IMXRT-Boards/50-Board-Control/uart_loopback.py:21]:
```suggestion
```

**Review by iabdalkader — CHANGES_REQUESTED** [review]
Can you please combine `pin_read.py` and `pin_write.py` in a simple `pin_control.py` example like other boards ? Let's keep those examples basic.

File: scripts/examples/50-OpenMV-Boards/50-IMXRT-Boards/50-Board-Control/spi_control.py (hunk 1 above)
Comment (kwagyeman) [reply scripts/examples/50-OpenMV-Boards/50-IMXRT-Boards/50-Board-Control/spi_control.py:9]: Fixed

File: scripts/examples/50-OpenMV-Boards/50-IMXRT-Boards/50-Board-Control/spi_loopback.py (hunk 2 above)
Comment (kwagyeman) [reply scripts/examples/50-OpenMV-Boards/50-IMXRT-Boards/50-Board-Control/spi_loopback.py:27]: Done

File: scripts/examples/50-OpenMV-Boards/50-IMXRT-Boards/50-Board-Control/uart_loopback.py (hunk 3 above)
Comment (kwagyeman) [reply scripts/examples/50-OpenMV-Boards/50-IMXRT-Boards/50-Board-Control/uart_loopback.py:21]: Done

**Comment by kwagyeman** [comment]
@iabdalkader Updated.

**Review by iabdalkader — APPROVED** [review]

**Pull request merged**